src/ray/raylet/scheduling/local_task_manager_interface.h

```diff
@@ -71,5 +71,63 @@ class ILocalTaskManager {
   virtual size_t GetNumWaitingTaskSpilled() const = 0;
   virtual size_t GetNumUnschedulableTaskSpilled() const = 0;
 };
+
+/// Dummy local task manager, which is used by gcs node only.
+class DummyLocalTaskManager : public ILocalTaskManager {
+ public:
+  DummyLocalTaskManager() {}
+
+  /// Queue task and schedule.
+  void QueueAndScheduleTask(std::shared_ptr<internal::Work> work) override {}
```

Comment:
should we RAY_CHECK to ensure `QueueAndScheduleTask` is never called?

Reply:
Sure.